app/models/person.rb

```diff
@@ -22,6 +22,7 @@ class Person < VersionedModel
   has_many :profiles, dependent: :destroy
   has_many :moves, through: :profiles
   has_many :person_escort_records, through: :profiles
+  has_many :youth_risk_assessments, through: :profiles
```

Comment:
Through moves instead?

Reply:
discussed in person: this should remain the same